announcement/angular-14 (#42)
SoftwareAG/c8y-releasenotes: content/release-10-14-0/announcements-10-14-0.md

### SDK changes

#### Planned

> As of release 10.16 we plan to update the Web SDK to Angular 14. See the [Angular upgrade guide](https://v14.angular.io/guide/update-to-latest-version) for more information on what needs to be changed in your custom application. You can use our [default upgrade guide](https://cumulocity.com/guides/web/upgrade/#update-to-an-newer-version) to update the files provided by the Cumulocity cli tool.

Reviewer: ```suggestion
As of release 10.16 we will update the Cumulocity IoT Web SDK to Angular 14. See the [Angular upgrade guide](https://v14.angular.io/guide/update-to-latest-version) for more information on what needs to be changed in your custom applications. Refer to our [default upgrade instructions](https://cumulocity.com/guides/web/upgrade/#update-to-an-newer-version) to update the files provided by the Cumulocity IoT CLI tool.
```
